core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -211,6 +211,8 @@ public ServerMain(String[] args) throws Exception {
 
             System.gc();
 
+            log.advisory().$("enjoy").$();
```

Comment:
I would suggest we go with "QuestDB Instance restarted" or something similar. In long run "enjoy" in logs would confusing.

Reply:
"enjoy" is a very unlikely word to be used in technical logging, in fact this idea comes from a large production cluster of applications I used to work on. The benefits are:
- easy to memorize
- low probability of false positive

We could get used to using "enjoy" in context of "instance started".

I would argue that `"QuestDB Instance restarted` is harder to memorize: "(I)nstance" - capital 'I', 'started" or "restarted" ? etc.